packages/expo-notifications/README.md

```diff
@@ -1 +1,367 @@
+      console.log(notification);
+    });
+    return () => subscription.remove();
+  }, []);
+
+  return (
+    // Your app content
+  );
+}
+```
+
+### `addNotificationsDroppedListener(listener: () => void): void`
+
+Listeners registered by this method will be called whenever some notifications have been dropped by the server. Applicable only to Firebase Cloud Messaging which we use as notifications service on Android. It corresponds to `onDeletedMessages()` callback. [More information can be found in Firebase docs](https://firebase.google.com/docs/cloud-messaging/android/receive#override-ondeletedmessages).
+
+#### Arguments
+
+A single and required argument is a function–callback.
+
+#### Returns
+
+A [`Subscription`](#subscription) object representing the subscription of the provided listener.
+
+### `addNotificationResponseReceivedListener(listener: (event: NotificationResponse) => void): void`
+
+Listeners registered by this method will be called whenever a user interacts with a notification (eg. taps on it).
+
+#### Arguments
+
+A single and required argument is a function accepting notification response ([`NotificationResponse`](#notificationresponse)) as an argument.
+
+#### Returns
+
+A [`Subscription`](#subscription) object representing the subscription of the provided listener.
+
+#### Examples
+
+Registering a notification listener using a React hook
+
+```tsx
+import React from 'react';
+import { Linking } from 'react-native';
+import * as Notifications from 'expo-notifications';
+
+export default function Container({ navigation }) {
+  React.useEffect(() => {
+    const subscription = Notifications.addNotificationResponseReceivedListener(response => {
+      const url = response.notification.request.content.data.url;
+      Linking.openUrl(url);
+    });
+    return () => subscription.remove();
+  }, [navigation]);
+
+  return (
+    // Your app content
+  );
+}
+```
+
+### `removeNotificationSubscription(subscription: Subscription): void`
+
+Removes a notification subscription returned by a `addNotification*Listener` call.
+
+#### Arguments
+
+A single and required argument is a subscription returned by `addNotification*Listener`.
+
+### `removeAllNotificationListeners(): void`
+
+Removes all notification subscriptions that may have been registered with `addNotification*Listener`.
+
+### `setNotificationHandler(handler: NotificationHandler | null): void`
+
+When a notification is received while the app is running, using this function you can set a callback that will decide whether the notification should be shown to the user or not.
+
+When a notification is received, `handleNotification` is called with the incoming notification as an argument. The function should respond with a behavior object within 3 seconds, otherwise the notification will be discarded. If the notification is handled successfully, `handleSuccess` is called with the identifier of the notification, otherwise (or on timeout) `handleError` will be called.
+
```

Comment:
we should also document here what the default behavior is if you don't set a notification handler